### PR TITLE
Add travis build

### PR DESCRIPTION
**Pull request by ruseinov**
@alpe FYI This repo needs to be enabled in Travis first, I've made a request through Travis UI

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+script:
+  - make test;
+  - if [[ "$TRAVIS_GO_VERSION" == "$MAIN_GO_VERSION" && "$TRAVIS_OS_NAME" == "linux" ]]; then
+    release_latest=$( [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_TAG" == "" && "$TRAVIS_PULL_REQUEST_BRANCH" == "" ]] && echo "yes" || echo "no" );
+    release_tag=$( [[ "$TRAVIS_TAG" != "" ]] && echo "yes" || echo "no" );
+
+    if [[ $release_latest == "yes" ]]; then
+    docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
+    BUILD_VERSION="latest" make docker
```
Comment (alpe) [inline .travis.yml]: req: this won't work as expected as you are setting the `v` prefix in https://github.com/iov-one/chatbot/blob/master/Makefile#L3

Comment (alpe) [reply .travis.yml]: I have also noticed that you push to `iov1` repo on docker hub. I think `iov1ops` is a better fit as this is not any of our products.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    docker logout;
+    fi;
+
+    if [[ $release_tag == "yes" ]]; then
+    docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
+    BUILD_VERSION="$TRAVIS_TAG" make docker
```
Comment (alpe) [inline .travis.yml]: please note that you have have the `v` in the tag name.

File: .travis.yml (hunk 1 above)
Comment (ruseinov) [reply .travis.yml]: Yeah, it makes sense I guess, will fix that

File: .travis.yml (hunk 2 above)
Comment (ruseinov) [reply .travis.yml]: good catch!

Comment (ruseinov) [reply .travis.yml]: I think I will remove the v from the Makefile to make it easier

**Pull request merged**